x-pack/legacy/plugins/maps/public/layers/sources/es_term_source.js

```diff
@@ -118,30 +124,6 @@ export class ESTermSource extends AbstractESAggSource {
     return false;
   }
 
-  _getRequestDescription(leftSourceName, leftFieldName) {
-    const metrics = this.getMetricFields().map(esAggMetric => esAggMetric.getRequestDescription());
```

Comment:
Hard to see in the diffs, but I removed `getRequestDescription` from ESAggField because it provided no value. So I just simplified the description for the _search request.